utils/kata-manager.sh

```diff
@@ -782,6 +821,25 @@ handle_docker()
 	docker --version
 }
 
+handle_nerdctl()
+{
+	{ containerd_installed; ret=$?; } || true
+	if [ "$ret" -eq 0 ]
+	then
+		info "Backing up previous $containerd_project configuration"
+		[ -e "$containerd_config" ] && sudo mv $containerd_config $containerd_config.system-$(date -Iseconds)
+	fi
+
+	install_nerdctl
+
```

Comment:
Since we're not calling `install_containerd()` (as nerdctl supplies its own version of containerd), we're not creating the sym-links for `ctr` and `containerd`. I guess we _probably_ don't need `ctr` since if we did, why are we installing `nerdctl`? :) But we could still benefit from the `containerd` sym-link, and one for `nerdctl` since we're assuming currently that `/usr/local/bin` is in the `$PATH`.

Related to this, and an alternative: We could update